src/main/java/dev/codesoapbox/dummy4j/dummies/DateAndTimeDummy.java

```diff
@@ -78,27 +99,39 @@ private long nextLongWithNegativeBounds(long lowerBound, long upperBound) {
         return dummy4j.number().nextLong(0, diff) + lowerBound;
     }
 
+    /**
+     * Generates a random date and time from the past that does not exceed the specified time lapse
```

Comment:
`that does not exceed the specified time lapse` → `that fits in the time range between {@code atMost units} and now`.

Or something like that.